bcda/models/models.go

```diff
@@ -80,10 +81,22 @@ type ACOBeneficiary struct {
 	// Join model needed for additional fields later, e.g., AttributionDate
 }
 
+type CCLF9 struct {
+	gorm.Model
+	CurrentNum   string `json:"current_number"`
+	PrevNum      string `json:"previous_number"`
+	PrevsEfctDt  string `json:"effective_date"`
+	PrevsObsltDt string `json:"obsolete_date"`
+}
+
 func (*ACOBeneficiary) TableName() string {
 	return "acos_beneficiaries"
 }
 
+func (*CCLF9) TableName() string {
```

Comment:
Is this function used?  Should we be doing this on other models?

Reply:
not sure about other models but gorm kept screwing up the table name by putting in an underscore

Reply:
Without it I'm pretty sure it tries to pluralize automatically, so it wasn't getting this one right